backend/app/api/endpoints/admin/public_bots.py

```
@@ -230,27 +480,118 @@ async def create_public_bot(
+    elif bot_data.shell_name:
+        # Form data mode - auto-create Ghost and optionally Model
+        logger.info(f"Creating public bot '{bot_data.name}' in form data mode")
+
+        # Validate shell exists as public resource
+        shell = (
+            db.query(Kind)
+            .filter(
+                Kind.user_id == 0,
+                Kind.kind == "Shell",
+                Kind.name == bot_data.shell_name,
+                Kind.is_active == True,
```

Ruff (0.14.14)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)